### PR TITLE
TC-666 Convert pricePerUnit tests to RTL

### PR DESCRIPTION
**Pull request by kscoulter**
Adds RTL test suite to pricePerUnit component.

### CODE REVIEW

File: solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js [hunk 1]
```diff
@@ -1,89 +1,74 @@
...
-  }
-
-  afterEach(() => {
-    jest.clearAllMocks()
+  beforeEach(() => {
+    render(<BreakEvenCalculator />)
+    const totalFixedCostYesRadioText = 'yes, I know the total of my monthly fixed costs'
+    userEvent.click(screen.getByLabelText(totalFixedCostYesRadioText))
+    userEvent.type(screen.getByLabelText('total fixed cost'), '1000')
+    fireEvent.submit(screen.getByTestId('fixedCosts-form'))
```
Comment (peggles2) [inline solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js:13]: Clicking on the continue button is more close to how the user interacts with the website. So shouldn't we change this to click instead?

File: solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js [hunk 2]
```diff
@@ -1,89 +1,74 @@
...
   })
 
-  it('returns an error if field is empty', () => {
-    const wrapper = shallow(
-      <PricePerUnit {...baseProps} />
-    )
+  test('resets app on restart analysis link', () => {
+    userEvent.click(screen.getByRole('button', { name: /restart analysis/i }))
+    screen.getByRole('heading', {name: /calculate your total fixed costs/i})
+    expect(screen.queryByRole('button', { name: /continue/i })).toBeNull()
```
Comment (peggles2) [inline solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js:46]: nice job checking for the button

File: solution/ui/src/components/organisms/unitSales/unitSales.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const UnitSales = (props) => {
   }, [props.visible])
   
   return (
-    <div className={`unitSales-container ${props.visible ? '' : 'hidden'}`} aria-hidden={!props.visible}>
+    <div aria-hidden={!props.visible} className={`unitSales-container ${props.visible ? '' : 'hidden'}`}>
```
Comment (peggles2) [inline solution/ui/src/components/organisms/unitSales/unitSales.js:36]: this should already be on master?

**Review by peggles2 — CHANGES_REQUESTED** [review]
Looks good, one change.

File: solution/ui/src/components/organisms/unitSales/unitSales.js (hunk 3 above)
Comment (kscoulter) [reply solution/ui/src/components/organisms/unitSales/unitSales.js:36]: yeah, it's just in a different spot

File: solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js (hunk 1 above)
Comment (kscoulter) [reply solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js:13]: Either way, this is not the part that's getting tested anyway. This is just to get to the second page.

Comment (peggles2) [reply solution/ui/src/components/organisms/pricePerUnit/pricePerUnit.test.js:13]: Right but the user gets to the second page by clicking 'continue'. So it's better to test it the way the user is going to be using it.

**Review by peggles2 — APPROVED** [review]
LGTM 👍

**Pull request merged**